src/core/mantle/data/csv.ts

```diff
@@ -0,0 +1,154 @@
+import type { Feature as GeoJsonFeature, Point } from "geojson";
+
+import type { Data, DataRange, Feature } from "../types";
+
+import { f, generateRandomString } from "./utils";
+
+export async function fetchCSV(data: Data, range?: DataRange): Promise<Feature[] | void> {
+  if (!data.url) {
+    const value = data.value;
+    const feature = makeFeature(value, range);
+    return feature ? [feature] : undefined;
+  }
+  const csvText = await (await f(data.url)).text();
+  return await parseCSV(csvText, data.csv, range);
+}
+
+const parseCSV = async (
```

Comment:
This is something that hasn't been maintained well on the frontend, but can we keep naming consistent for abbreviations? 
Since reearth-web is already kinda a mess with this, maybe you can make a decision on whether we do `Url` or `URL`, `Csv` or `CSV`, `geoJSON` or `geoJson`, etc. and then we can make it a rule.

Reply:
I agree. Which do you prefer whether upper case(like `CSV`) or title case(like `Csv`)?
I will fix the naming in this PR first with following your opinion.

Reply:
Seems like a all uppercase or all lowercase is relatively favoured ([airbnb's style guide](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms)
I think it makes sense and will rarely become a problem (ie. with multiple abbreviations side-by-side

Reply:
It's make sense. I will fix it.
Thank you:pray:

Reply:
I renamed some abbreviations.
[refactor: rename abbreviations](https://github.com/reearth/reearth-web/pull/382/commits/a5891a6d2c2d536d9fe0840913aed62ad471aca5)

Reply:
Sorry you missed `makeGeoJsonFromArray`. Should be `makeGeoJSONFromArray`, right?
Also, in your refactor, you updated the geojson part, but not the csv part (should be `CSVGeoJSONFeature`, not `CsvGeoJSONFeature`

Reply:
Oops... I'm sorry it's my mistakes...
[chore: rename some abbreviation](https://github.com/reearth/reearth-web/pull/382/commits/28cee63fd90c967fe0bd33fed2d7d9532655d0d0)